content/SCALE/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -135,7 +135,7 @@ Notable changes:
 
 ### 25.04-BETA.1 Known Issues
 
-* 
+* An issue has been discovered for cloud sync tasks configured with **Filename Encryption**, which is available in **Advanced Remote Options** ([NAS-132472](https://ixsystems.atlassian.net/browse/NAS-132472)). As this is an upstream issue in rclone, we recommend that users should not create new cloud sync tasks with **Filename Encryption** enabled. Existing users of this feature must leave it enabled for existing cloud sync tasks to be able to recover backups.
```

Comment:
I missed this the first time, in this case you are talking about the feature, like saying  replication, so Filename Encryptions as the UI element is not correct here unless specifically talking about this in context, this is more a general mention. so I recommend either of these options:
```suggestion
* An issue has been discovered for cloud sync tasks configured using the **Filename Encryption** setting, which is available in **Advanced Remote Options** ([NAS-132472](https://ixsystems.atlassian.net/browse/NAS-132472)). As this is an upstream issue in rclone, we recommend that users should not create new cloud sync tasks with the **Filename Encryption** setting enabled. Existing users of this feature must leave it enabled for existing cloud sync tasks to be able to recover backups.
```
or
```suggestion
* An issue has been discovered for cloud sync tasks configured with file name encryption, which is available in **Advanced Remote Options** ([NAS-132472](https://ixsystems.atlassian.net/browse/NAS-132472)). As this is an upstream issue in rclone, we recommend that users should not create new cloud sync tasks with the **Filename Encryption** setting enabled. Existing users of this feature must leave it enabled for existing cloud sync tasks to be able to recover backups.
```